### PR TITLE
feat(general): MP-290 Create Module, feat(general): MP-291 Create Application

### PR DESCRIPTION
**Pull request by args06**
## Update Items
- [x] [[MP-290](https://args06.atlassian.net/browse/MP-290)] Create Module
- [x] [[MP-291](https://args06.atlassian.net/browse/MP-291)] Create Application

## Summary Description
- Create Application Class
- Create a module for Hilt Injection

## *Type of Change
- [ ] docs: Documentation only changes, README.md, Wiki
- [x] feat: A new feature
- [ ] refactor: A code change that neither fixes a bug nor adds a feature
- [ ] test: Anything related to unit/integration testing
- [ ] revert: Revert a commit

## Type of Scope
- [ ] album: Changes that affect user packages
- [ ] music: Changes that affect music packages
- [ ] account: Changes that affect account packages
- [ ] favorite: Changes that affect favorite packages
- [ ] playlist: Changes that affect playlist packages
- [ ] playlist-detail: Changes that affect playlist-detail packages
- [ ] music-artist: Changes that affect music-artist packages
- [x] general: all stuff except above

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -150,4 +151,7 @@ dependencies {
 
     // Splash Screen
     implementation("androidx.core:core-splashscreen:$splashScreenVersion")
+
+    implementation "com.google.dagger:hilt-android:2.44"
+    kapt "com.google.dagger:hilt-android-compiler:2.44"
```
Comment (RTAgung) [inline app/build.gradle]: use variable for version dependencies

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -2,6 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```
Comment (RTAgung) [inline app/src/main/AndroidManifest.xml:7]: what is the purpose of this?

**Review by RTAgung — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (args06) [reply app/src/main/AndroidManifest.xml:7]: image
It's say required

File: app/build.gradle (hunk 1 above)
Comment (args06) [reply app/build.gradle]: okay

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (RTAgung) [reply app/src/main/AndroidManifest.xml:7]: okay

**Review by RTAgung — APPROVED** [review]

**Pull request merged**